README.md

```diff
@@ -7,10 +7,6 @@ The Feature Hub is an opinionated JavaScript implementation of the micro
 frontends approach to creating scalable web applications with multiple teams and
 different technologies.
 
-**Note:** This software is in active development and is subject to potentially
-breaking changes. It is not yet used in production. The currently planned
-milestones can be viewed [here][milestones].
```

Comment:
Maybe we should delete this comment in a commit that contains the words `BREAKING CHANGE` to trigger the 1.0 release?